### ISSUE TITLE
how to fix cookies from getting expired in few days. i need help on this guys 

### ISSUE THREAD

**Issue by henzo5050** [kgretzky/evilginx2#757]

**Comment by 456478** [comment]
log in via the cookies, do a random action resulting in a request i.e. refresh the cookies, repeat
that being said, everything can be automated. look into go-rod, it's pretty solid

**Comment by henzo5050** [comment]
> log in via the cookies, do a random action resulting in a request i.e. refresh the cookies, repeat that being said, everything can be automated. look into go-rod, it's pretty solid

ok thanks. but will this make the cookies not expire in few days

**Comment by henzo5050** [comment]
> log in via the cookies, do a random action resulting in a request i.e. refresh the cookies, repeat that being said, everything can be automated. look into go-rod, it's pretty solid

can you be kind enough to show me how this works. probably a step to step to help prevent captured cookies from getting expired in few days

**Comment by henzo5050** [comment]
please

**Comment by 456478** [comment]
Yes, that would keep the cookies fresh and prevent them from expiring. Example of how to set and read cookies with go-rod: https://github.com/go-rod/rod/blob/bc44c39c9b4352c15d00bef6f6a1071205d2c388/lib/examples/compare-chromedp/cookie/main.go#L17

**Comment by Support-1535** [comment]
Hello! If you were already able to resolve your doubts and achieve your goals, close the issue so that we know which ones are pending.

Thank you!

If you want expedited help and have the assistance of many professional people, join our private group. Contact us support@evilginx2.com to give you access to the group.
